projects/cdk/directives/fullscreen/fullscreen.directive.ts

```diff
@@ -0,0 +1,67 @@
+import {DOCUMENT} from '@angular/common';
+import {
+    ChangeDetectionStrategy,
+    Component,
+    ElementRef,
+    EventEmitter,
+    inject,
+    Input,
+    Output,
+    signal,
+    ViewChild,
+    ViewEncapsulation,
+} from '@angular/core';
+import {TuiRoot} from '@taiga-ui/core/components/root';
+
+@Component({
```

Comment:
Let's move it to components and to core, since it relies on tui-root

Reply:
fixed